### ISSUE TITLE
Cannot execute squid unit tests - missing data

### ISSUE THREAD

**Issue by Bertk** [SonarOpenCommunity/sonar-cxx#1699]
I tried to build the plug-in locally but the unit tests failed because of not existing hard coded path `D:\Development\\Source\Cpp\Dummy\\src\\main.cpp`

- VM using Windows 10 (disk drive C:\, dvd drive D:\)
- Maven 3.5.4
- Java version: 11.0.2

```java

    private static final String VC_CHARSET = "UTF8";

    public static final String OVERALLINCLUDEKEY = "CxxOverallInclude";
    public static final String OVERALLDEFINEKEY = "CxxOverallDefine";
    public static final String REFERENCE_DETAILED_LOG = "src/test/resources/logfile/msbuild-detailed-en.txt";
    public static final String UNIQUE_FILE = "D:\\Development\\Source\\Cpp\\Dummy\\src\\main.cpp";
```

### Description

A test uses full path of not existing folder.

### Steps to reproduce the problem

use system with single drive.

### Expected behavior

test is executed without error and all resources are available in the git repository or folders are created before test is started.

### Actual behavior

Test fails.

### Known workarounds

disable test which uses not existing path

### LOG file

![failed-unit test](https://user-images.githubusercontent.com/2315215/55223780-43621800-520f-11e9-9c86-36717d77da34.png)

**Comment by guwirth** [comment]
Hi @Bertk,

agree there should be no absolute paths in the tests. On the other hand it's working fine on our CI (AppVeyor) and also on my local PC. Maybe a problem that D is a DVD drive in your test environment.

Regards,

**Comment by guwirth** [comment]
Closed as part of #1726
